site/docs/ch/roadmap.md

```diff
@@ -0,0 +1,35 @@
+# Roadmap
+
+Arctic 预计每2-3个月发布一个大版本，每1个月发布一个小版本，每个版本都会有一个发布计划，发布计划会在 [Github Milestones](https://github.com/NetEase/arctic/milestones) 中发布。
+
+## 2022年发布计划
+
+| 版本                                                     | 发布日期          |
+| --------------------------------------------------------| ---------------- |
+| [0.3.1](https://github.com/NetEase/arctic/milestone/1)  | 2022.08          |
+| [0.3.2](https://github.com/NetEase/arctic/milestone/2)  | 2022.09          |
+| [0.4.0](https://github.com/NetEase/arctic/milestone/3)  | 2022.11          |
+| [0.5.0](https://github.com/NetEase/arctic/milestone/4)  | 2022.12          |
+
+## 开发中
+
+| 特性                                                     | 发布版本        |   跟踪  |
+| --------------------------------------------------------| -------------- |--------|
+|Use arctic table as a dimension table with Flink         | 0.3.1          | [ARCTIC-94](https://github.com/NetEase/arctic/issues/94) |
+|Support Flink 1.15 version                               | 0.3.1          | [ARCTIC-166](https://github.com/NetEase/arctic/issues/166) |
```

Comment:
What about supporting Flink 1.15, 1.16 version?

Reply:
As Flink 1.16 is not released now, I would like to add it to roadmap after its releasing.